tests/Core/Slider/FluentSliderTests.razor

```diff
@@ -77,19 +77,4 @@
         // Assert
         cut.Verify();
     }
-
-    [Fact]
-    public void FluentSlider_Binding()
```

Comment:
Why did you removed this test ?

Reply:
That was because of the revert action. I'll move this back in